src/modules/85-cv/components/Notifications/NotificationsContainer.utils.ts

```diff
@@ -147,6 +145,21 @@ export function validateNotificationConditions(
     if (!condition?.condition && !values?.conditions?.[index]?.condition) {
       set(errors, `conditions.${index}.condition`, `${getString('cv.notifications.validations.conditionIsRequired')}`)
     }
+    if (
+      condition?.condition?.value == Condition.CODE_ERRORS ||
+      values?.conditions?.[index]?.condition == Condition.CODE_ERRORS
```

Comment:
An alternative way to write this would be using `Array.includes`
```suggestion
      [condition?.condition?.value, values?.conditions?.[index]?.condition].includes(Condition.CODE_ERRORS)
```

Reply:
That's cool, but not as obvious to read as `||` operator in my opinion.